addon-chrome/db.js

```diff
@@ -0,0 +1,118 @@
+const DB_NAME = "activitystream";
```

Comment:
Give more thought to db upgrades. Reality of designing a store is that you will need to upgrade your schema at some point. Currently your "schema" is passed in (set of keyStores), but you're implying a version number internally. This doesn't seem right. You should be versioning your schema, and keeping a version number close to your schema. You should be explicitly handling version numbers in your upgrades. Your upgrade function should have different paths for "creating a fresh db" and "upgrading from previous versions".

Do you need to migrate data as well during schema updates? Currently you're dropping everything when version changes.

Also, nit: I'd simplify method names (e.g. getFromDb -> get)